.github/workflows/ci-workflow.yml

```diff
@@ -7,9 +7,7 @@ on:
       - master
       - develop
   pull_request:
-    branches:
-      - master
-      - develop
+    types: [opened, synchronize, reopened]
```

Comment:
C'est mieux que ce qu'il y avait avant ?

Reply:
The goal is to trigger the CI even if the target branch of the PR is not develop
My opinion is that if it's a PR it needs a CI
It's useful for me as I opened two newer PRs that cascade into each other